lib/feedjira/parser/podcast.rb

```diff
@@ -50,6 +48,10 @@ class Podcast
       elements :link, as: :hubs, value: :href, with: {rel: "hub"}
       elements :"atom10:link", as: :hubs, value: :href, with: {rel: "hub"}
 
+      element :link, as: :feed_url, value: :href, with: {rel: "self"}
+      element :"atom10:link", as: :feed_url, value: :href, with: {rel: "self"}
+      element :"atom:link", as: :feed_url, value: :href, with: {rel: "self"}
```

Comment:
That `attr_accessor` was apparently doing nothing?  I pulled it down here (copying some FeedJira other parsers) to look for any "self" link in the RSS.

Reply:
what is the FeedJira library and what do these `elements` do?

Reply:
FeedJira is an RSS parsing gem - and these `element` / `elements` are part of its [syntax](https://github.com/feedjira/feedjira#adding-attributes-to-only-one-class).

I sort of pieced these together from the other parts of this parser file, and looking at how [the builtin Atom namespace parser](https://github.com/feedjira/feedjira/blob/main/lib/feedjira/parser/atom.rb#L13) works.